internal/connectors/client/connector.go

```diff
@@ -179,3 +180,27 @@ func (d *ClientYdbConnector) ForgetOperation(ctx context.Context, clientDb *ydb.
 
 	return response, nil
 }
+
+func (d *ClientYdbConnector) CancelOperation(ctx context.Context, clientDb *ydb.Driver, operationId string) (*Ydb_Operations.CancelOperationResponse, error) {
+	if clientDb == nil {
+		return nil, fmt.Errorf("unititialized client db driver")
+	}
+
+	client := Ydb_Operation_V1.NewOperationServiceClient(ydb.GRPCConn(clientDb))
+	xlog.Info(ctx, "Cancelling operation",
+		zap.String("id", operationId),
+	)
+
+	response, err := client.CancelOperation(
+		ctx,
+		&Ydb_Operations.CancelOperationRequest{
+			Id: operationId,
+		},
+	)
+
+	if err != nil {
+		return nil, fmt.Errorf("error cancelling operation: %s", err.Error())
```

Comment:
I'm not sure if we can see this error in the server log?
Let's log errors in grpc handlers